2-mandatory/1-writers.js

```diff
@@ -72,6 +76,13 @@ Exercise 2:
 
```

Comment:
A few points on this one:
1. What happens if we had a 38-year-old author who was not alive? Would this could still work?
2. Whenever you see something like `writer.alive === false` in your code, you can rewrite it to `!writer.alive`. Can you think of why that is?
3. Now that you have this working with a `for` loop, can you rewrite this using array methods (maybe `filter` and `forEach`)?